### PR TITLE
Component structure

### PR DESCRIPTION
**Pull request by khumair**

### CODE REVIEW

**Review by Tsaikkari — CHANGES_REQUESTED** [review]
I've already imported Poppins font family as a link from google fonts in the html file, so remove all these. They take space.

File: src/components/companyMatchPage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            <div className="card shadow">
+              <div className="card-body">
+                <div className="row">
+                  <div className="col-xs-1 col-sm-2 col-md-1 mr-3">
+                    <Image src={male} />
+                    <br></br>
+                  </div>
+                  <div className="col-xs-8 col-sm-7 col-md-8">
+                    <h5>Car System Alerts</h5>
+                    <p>Check the battery and alternator conditions</p>
```
Comment (Tsaikkari) [inline src/components/companyMatchPage/index.tsx]: Use real attributes like name. 
Add the Posts, Chats, and Match links and the View and Chat buttons into the page.

File: src/components/companyProfile/index.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Form, Col, Row, Container, Button, Image } from "react-bootstrap";
+import uploadImage from "../../assets/images/upload-Image.png";
+const CompanyProfile = () => {
+  return (
+    <Container fluid="md">
+      <h2 className="row justify-content-center">Company Profile</h2>
+      <Row>
+        <Col xs>
```
Comment (Tsaikkari) [inline src/components/companyProfile/index.tsx:9]: xs="12" ?

File: src/components/companyProfile/index.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            </Form.Group>
+
+            <Form.Group as={Row} controlId="formElement">
+              <Form.Label column sm="4">
+                city City
+              </Form.Label>
+              <Col sm="8">
+                <Form.Control type="text" placeholder="Password" />
+              </Col>
+            </Form.Group>
```
Comment (Tsaikkari) [inline src/components/companyProfile/index.tsx]: Why password?

File: src/scss/fonts.scss [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+      url("../assets/font/poppins/poppins-v5-latin-300.woff") format("woff"),
+    /* Modern Browsers */ url("../assets/font/poppins/poppins-v5-latin-300.ttf")
+      format("truetype"),
+    /* Safari, Android, iOS */
+      url("../assets/font/poppins/poppins-v5-latin-300.svg#Poppins")
+      format("svg"); /* Legacy iOS */
+}
+/* poppins-300italic - latin */
+@font-face {
+  font-family: "Poppins";
```
Comment (Tsaikkari) [inline src/scss/fonts.scss:22]: Remove all these font-faces.

**Review by Tsaikkari — CHANGES_REQUESTED** [review]

File: src/scss/fonts.scss (hunk 4 above)
Comment (khumair) [reply src/scss/fonts.scss:22]: removed

Comment (Tsaikkari) [reply src/scss/fonts.scss:22]: Please remove the font file in your src/assets folder.

**Pull request closed without merging**